JavaScript/chapter01/1.5 - OneAway/rroque98_sol.js

```diff
@@ -0,0 +1,43 @@
+/* There are 3 types of edits that can be made on a string:
+insert character, remove character, replace a character
+Given 2 strings write a function that will check if it is
+1 or 0 edits away*/
+
+const isOneAway = (str1, str2) => {
+  if (str1.length === str2.length) {
+    let errorCount = 0;
+    for (let i = 0; i < str1.length; i++) {
+      if (str1[i] !== str2[i]) {
+        errorCount++;
+      }
+      if (errorCount > 1) {
+        return false;
+      }
+    }
+  } else {
+    let errorCount = 0;
+    const longestStr = findLongestStr(str1, str2);
+    let x = 0;
+    for(let i = 0; i < longestStr; i++) {
```

Comment:
this loop assumes that the lengths of the strings differe by at most 1, but it's not necessarily true. it might be better to actually check that explicitly.

Reply:
Here, I also meant to loop through longestStr.length. I must have been a bit tired when I worked on this problem. Thanks!